packages/query/src/utils/logger.ts

```diff
@@ -36,3 +38,44 @@ export class NestLogger implements LoggerService {
     this.logger.warn(message);
   }
 }
+
+export const PinoConfig = {
+  logger: getLogger('express'),
+  serializers: {
+    req(req) {
+      const body = req.raw.body;
+      if ('operationName' in body && body.query) {
+        // Logging IntrospectionQuery payload clutters logs and isn't useful
+        if (body.operationName !== 'IntrospectionQuery') {
+          req.payload = stringify(
+            gql`
+              ${body.query}
+            `
+          );
+        }
+      }
+      return req;
+    },
+    res(res) {
+      if (res.headers.stack) {
+        res.stack = res.headers.stack;
+        delete res.headers.stack;
+      }
+      if (res.headers.message) {
+        delete res.headers.message;
```

Comment:
Message in the headers is still not removed. check browser's network inspect

Reply:
removed